packages/bvaughn-architecture-demo/components/console/Focuser.test.tsx

```diff
@@ -0,0 +1,57 @@
+import { fireEvent, screen } from "@testing-library/react";
+
+import { renderFocused } from "../../src/utils/testing";
+
+import Focuser from "./Focuser";
+
+describe("Focuser", () => {
+  it("should render the with no focus region", async () => {
+    await renderFocused(<Focuser />, {
+      focusContext: {
+        range: null,
+        rangeForDisplay: null,
+      },
+      sessionContext: {
+        duration: 60_000,
+      },
+    });
+
+    expect(await screen.queryByText("Focus off")).toBeInTheDocument();
+    expect(await screen.queryByText("0:00 – 1:00")).toBeInTheDocument();
+  });
```

Comment:
This is an example of the approach we might use to render a small part of our app to test in isolation. The `renderFocused` helper method could be shared between all tests. It accepts optional params which can be used to define a test scenario.